common/lib/xmodule/xmodule/library_root_xblock.py

```diff
@@ -29,7 +29,8 @@ class LibraryRoot(XBlock):
     advanced_modules = List(
         display_name=_("Advanced Module List"),
         help=_("Enter the names of the advanced components to use in your library."),
-        scope=Scope.settings
+        scope=Scope.settings,
+        xml_node=True,
```

Comment:
The export code should be encoding lists/dicts into fields, but I agree that it's better in this case to just use `xml_node=True`.